tests/xbot/common/properties/PropertyTest.java

```diff
@@ -106,9 +108,12 @@ public void testLoadingValue() {
     
     @Test
     public void testLoadingValueAfterCreation() {
-    	((MockPermanentStorage)propertyManager.permanentStore).addTestDouble("speed", 0.5);
-    	((MockPermanentStorage)propertyManager.permanentStore).addTestBoolean("isTrue", true);
-    	((MockPermanentStorage)propertyManager.permanentStore).addTestString("string", "teststring");
+    	   	
+    	propertyManager.createProperty("speed", 0.5);
+        propertyManager.createProperty("isTrue", true);
+        propertyManager.createProperty("string", "teststring");
+    	
```

Comment:
The issue is that there are mixed tabs and spaces; tabs aren't quite as long as four spaces in this instance